sql/core/src/test/resources/sql-tests/inputs/number-format.sql

```diff
@@ -0,0 +1,13 @@
+-- Verifies how we parse numbers
+
+-- parse as ints
+select 1, -1;
```

Comment:
Spark SQL does not yet have a great strategy for this. I think we can have a small set of predefined tables, and then create tables on the fly using "create table x as select 1, 2, 3 union all select 2, 3, 4".

We will likely need to iterate on this.